.github/workflows/pull-request.yml

```diff
@@ -183,6 +183,13 @@ jobs:
           # FIXME: Chrome really doesn't seem to like yew-stdweb
           wasm-pack test --firefox --headless -- --features "wasm_test httpbin_test"
 
+      - name: Run tests - yew-services
```

Comment:
Did all the httpbin tests move to yew-services? I think we can remove the httpbin feature if that's the case

Reply:
Yep, it was unused. I removed it,